.github/workflows/go-check.yml

```diff
@@ -14,19 +15,88 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  go-check:
-    uses: ipdxco/unified-github-workflows/.github/workflows/go-check.yml@v1.0.22
+  go-check-all:
+    name: go-check / All
+    runs-on: ubuntu-latest
+    services:
+      mysql:
+        image: mysql:8.0
+        env:
+          MYSQL_ROOT_PASSWORD: root
+          MYSQL_DATABASE: singularity
+          MYSQL_USER: singularity
+          MYSQL_PASSWORD: singularity
+        ports:
+          - 3306:3306
+        options: >-
+          --health-cmd="mysqladmin ping"
+          --health-interval=10s
+          --health-timeout=5s
+          --health-retries=3
+      
+      postgres:
+        image: postgres:15
+        env:
+          POSTGRES_USER: singularity
+          POSTGRES_PASSWORD: singularity
+          POSTGRES_DB: singularity
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    
+    steps:
+      - uses: actions/checkout@v3
+      
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: "1.21.x"
+          
+      - name: Wait for PostgreSQL
+        run: |
+          echo "Waiting for PostgreSQL..."
+          for i in {1..10}; do
+            if PGPASSWORD=singularity psql -h localhost -U singularity -d singularity -c "SELECT 1" > /dev/null 2>&1; then
+              echo "Postgres is ready!"
+              break
+            fi
+            sleep 3
+          done
+
+      - name: Verify MySQL connection
+        run: mysql -h127.0.0.1 -P3306 -usingularity -psingularity -e "SELECT VERSION();"
+
+      - name: Verify PostgreSQL connection
+        run: PGPASSWORD=singularity psql -h localhost -U singularity -d singularity -c "SELECT version();"
```

Comment:
Why do you create a MySQL connection and a PSQL connection? Singularity seems to support both, so I could understand wanting to test with both, but I don't see you passing a connection string to Singularity anywhere so I think this test is still using sqllite. Have I missed where you configure the database you set up for testing?

https://data-programs.gitbook.io/singularity/installation/deploy-to-production